uber/templates/jobs/main_menu.html

```diff
@@ -14,7 +14,7 @@
     </td>
     <td width="50%">
         <select id="page" style="font-size:14pt ; font-weight:bold" onChange="mainMenuDropdownChanged()">
```

Comment:
Ahhh dang, I knew I was probably doing it the inefficient way. Let me fix that. :)